src/js/events/CosmosPackagesActions.js

```diff
@@ -149,14 +187,65 @@ const CosmosPackagesActions = {
     });
   },
 
+  fetchServiceDescription(serviceId) {
+    RequestUtil.json({
+      contentType: getContentType({
+        action: "describe",
+        actionType: "request",
+        entity: "service",
+        version: "v1"
+      }),
+      headers: {
+        Accept: getContentType({
+          action: "describe",
+          actionType: "response",
+          entity: "service",
+          version: "v1"
+        })
+      },
+      method: "POST",
+      url: `${Config.rootUrl}/cosmos/service/describe`,
+      data: JSON.stringify({ serviceId }),
```

Comment:
I will have to change this to { appId: serviceId}....the API expects the key name appId